lib/mail/meetings/update/direct-template.tsx

```diff
@@ -8,7 +8,9 @@ import {
   MeetingDisplay,
   P,
 } from 'lib/mail/components';
-import { Meeting, User } from 'lib/model';
+
```

Comment:
Why the empty space? There should be no empty space between import groups (see ESLint rules).